src/doc/trpl/patterns.md

```diff
@@ -70,8 +70,7 @@ This prints `something else`
 
 # Bindings
 
-If you’re matching multiple things, via a `|` or a `...`, you can bind
-the value to a name with `@`:
+You can bind values to names with `@`:
```

Comment:
Hmm, could we be more elaborate here, maybe? The point of `@` is that you can both bind the whole value and destructure it in the same pattern. If one only wanted to bind the value to a name, they could use an identifier.

See what I mean? I can't quite put it into words myself. :-)